tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleCore.java

```diff
@@ -154,9 +159,48 @@ private static void configureIfLiferayProject( final IProject project, final Gra
         {
             Job job = new WorkspaceJob( "Checking gradle configuration" )
             {
+
                 @Override
                 public IStatus runInWorkspace( IProgressMonitor monitor ) throws CoreException
                 {
+                    IFile bndFile = project.getFile( "bnd.bnd" );
+
+                    //case 1: has bnd file
+                    if( bndFile != null && bndFile.exists() )
```

Comment:
yeah , we already have the detection , see 
https://github.com/liferay/liferay-ide/blob/master/tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleCore.java#L153